common/src/main/java/com/wynntils/models/items/encoding/impl/block/EndDataTransformer.java

```diff
@@ -12,8 +12,6 @@
 import com.wynntils.utils.type.UnsignedByte;
 
 public class EndDataTransformer extends DataTransformer<EndData> {
-    public static final byte ID = (byte) 255;
```

Comment:
I kind of like having a systematic constant like this, rather than a hard-coded value in a getter. I think you should basically revert this commit, but make the ID fields private (and keep the fix in the registry).